### ISSUE TITLE
Moar tests!

### ISSUE THREAD

**Issue by drj11** [RSE-Sheffield/sheffield-covid-19#25]
Thanks to #21 we do now have some tests. However, more would always be welcome!

**Comment by muneerahp** [comment]
Happy to add more (and CI running of tests, can open this separately if here is messy) if nobody else wants to.

**Comment by drj11** [comment]
yeah, CI would be good. I was going to create an issue for running tests in CI but got distracted by a scheduled meeting. If something is obviously an improvement, just open a PR.
